introduction.md

```diff
@@ -0,0 +1 @@
+Hello My name is Suhas Deshpande and I am software Developer who loves Javascript and Git. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.